modules/exploits/windows/http/desktopcentral_deserialization.rb

```diff
@@ -83,26 +82,29 @@ def check
     )
 
     unless res
-      return CheckCode::Unknown('Target is not responding to check')
+      return CheckCode::Unknown('Target did not respond to check request.')
```

Comment:
Appreciate the English improvements here, approved!

Reply:
Aligned with newer modules!